MediaRecorder.bs

```diff
@@ -286,11 +276,36 @@ interface MediaRecorder : EventTarget {
     <li>Set |recorder|'s {{state}} to {{recording}}, and run the following steps
     in parallel:
     <ol>
-      <li>Start recording all tracks in |tracks| using the {{MediaRecorder}}'s
-      current configuration, and gather the data into a {{Blob}} <var>blob</var>
-      and queue a task, using the DOM manipulation task source, to
-      <a>fire an event</a> named <a>start</a> at <var>recorder</var>, then return
-      <code>undefined</code>.</li>
+      <li>Start recording all tracks in |tracks| using the |recorder|'s current
+      configuration and gather the data into a {{Blob}} <var>blob</var>. If the
+      codec or container has not yet been specified, the User Agent specifies
+      it. The User Agent MAY take the sources of the |tracks| into account when
+      making this decision as long as it does not violate the current
+      configuration.
```

Comment:
I'd make it "container or codec" since an application may constrain a container without constraining a codec, but not the other way around.

The are multiple codecs per recording, so "codec" in your text needs to be plural. There's at least one audio codec and one video codec when the MediaRecorder is recording such tracks.

I'd maybe make some other small changes like:
"If the container and codecs to use for the recording have not yet been fully specified, the User Agent specifies them in |recorder|'s current configuration. The User Agent MAY take the sources of the tracks in |tracks| into account when deciding which container and codecs to use."
This is mostly by personal preference, do with it as you wish.

Reply:
I updated the text to use your suggestion.